perfkitbenchmarker/benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -59,27 +59,27 @@
              '$FABAN_HOME/lib/fabandriver.jar:$JAVA_HOME/lib/tools.jar:'
              '$FABAN_HOME/search/build/lib/search.jar')
 
-flags.DEFINE_string('client_heap_size', '2g',
+flags.DEFINE_string('websearch_client_heap_size', '2g',
```

Comment:
I guess you could rename the benchmark to cloudsuite_websearch, that way, when you abbreviate flags to cs_websearch, it would be more consistent than previously.
If you go with faban/solr flag names, then ramp_up, steady_state and scale are also faban parameters, num_clients is number of machines the faban runs it's agents, but distribution belongs to benchmark configuration.
If cs_websearch for flags work for you, I think it is better option among those two options.